pyaedt/rmxprt.py

```diff
@@ -325,3 +329,69 @@ def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         setup.auto_update = True
         setup.update()
         return setup
+
+    @pyaedt_function_handler()
+    def export_configuration(self, output_file):
+        """Export Rmxprt project to config file.
+
+        Parameters
+        ----------
+        output_file : str
+            Full path to json file to be created.
+
+        Returns
+        -------
+        str
+           Full path to json file created.
+        """
+
+        def jsonalize(dict_in, dict_out):
+            dict_out[dict_in.node] = {}
+            for k, v in dict_in.props.items():
+                if not k.endswith("/Choices"):
+                    dict_out[dict_in.node][k] = v
+            for _, c in dict_in.children.items():
```

Comment:
why not taking directly the values?

```suggestion
            for c in dict_in.children.values():
```